rogii/environment_linux.cmake

```diff
@@ -0,0 +1,4 @@
+set(
+    REPOSITORY_OS
+    "linux"
+)
```

Comment:
We need gxx_runtime package for checking compiler under linux.
```
CNPM_ADD_PACKAGE(
    NAME
        gxx_runtime
    VERSION
        9.2.1
    BUILD_NUMBER
        0
)
```

Reply:
Added